actions/parse-push/src/main.ts

```diff
@@ -64,49 +70,59 @@ async function getSlug(commit: any): Promise<string> {
  *             Due to the nature of problem-specs, this will generally hold true.
  *
  * @returns The PR associated with the commit.
- * @param commit_sha
+ * @param commitSha
  */
-export async function getPullRequestHTMLURL(commit_sha: string): Promise<string> {
-    const octokit = gh.getOctokit(core.getInput('token'))
-    const pullCandidates = await octokit.repos.listPullRequestsAssociatedWithCommit({
-        owner: gh.context.repo.owner,
-        repo: gh.context.repo.repo,
-        commit_sha: commit_sha,
-        mediaType: {
-            previews: ['groot'], // This endpoint is part of a preview
-        },
-    })
+export async function getPullRequestHtmlUrl(
+    commitSha: string
+): Promise<string> {
```

Comment:
printWidth of 80 is way too short. Splitting this over 3 lines is not an improvement whatsoever.

Reply:
You can't "sometimes" use short lines. Using a larger width is _almost always_ a mistake and I agree with it.

https://prettier.io/docs/en/options.html#print-width

> For readability we recommend against using more than 80 characters:
>
> In code styleguides, maximum line length rules are often set to 100 or 120. However, when humans write code, they don’t strive to reach the maximum number of columns on every line. Developers often use whitespace to break up long lines for readability. In practice, the average line length often ends up well below the maximum.
>
> Prettier’s printWidth option does not work the same way. It is not the hard upper allowed line length limit. It is a way to say to Prettier roughly how long you’d like lines to be. Prettier will make both shorter and longer lines, but generally strive to meet the specified printWidth.
>
> Remember, computers are dumb. You need to explicitly tell them what to do, while humans can make their own (implicit) judgements, for example on when to break a line.
>
> In other words, don’t try to use printWidth as if it was ESLint’s max-len – they’re not the same. max-len just says what the maximum allowed line length is, but not what the generally preferred length is – which is what printWidth specifies.

Reply:
I'll counter with Linus: http://lkml.iu.edu/hypermail/linux/kernel/2005.3/08168.html

Reply:
That's not a counter-argument. `printWidth` doesn't do what you think it does.